paper-slider.html

```diff
@@ -113,6 +113,7 @@
       <paper-input id="input" class="slider-input" disabled$="[[disabled]]" on-change="_inputChange"></paper-input>
     </template>
 
+    <input type="hidden" value="[[value]]" name="[[name]]">
```

Comment:
I don't think you need to do this. If you just add the `IronFormElementBehaviour`, and set up name/value correctly, an `iron-form` will pick it up.

I don't think we need to assume that a native form should understand paper elements.